### ISSUE TITLE
Neuordnung und Verbesserung der JavaScript Dateien

### ISSUE THREAD

**Issue by FabiApfelkern** [socia-platform/htwplus#75]

**Comment by FabiApfelkern** [comment]
Erster Schritt für bessere JS ist gemacht. Ich habe alles etwas umsortiert und Bower eingeführt. Alle Vendor JS Libraries werden jetzt damit runtergeladen. Außer Bootstrap, dafür brauchen wir noch Grunt zum customizen. Das kommt später. Außerdem habe ich das sbt plugin JSHint installiert. Damit werden unsere JavaScripts auf Fehler geprüft. Ich habe auch direkt welche korrigiert in allen Dateien. Daher muss dieser Branch von jedem mal getestet werden. 
@IvenZ Wenn ich das in den dev merge muss auf dem Server node Bereit sein und die entsprechenden Kommandos ausgeführt werden.

**Comment by IvenZ** [comment]
wollen wir uns da mal zusammen setzen?

**Comment by FabiApfelkern** [comment]
Können wir machen. Ist aber nicht so schwierig.

**Comment by De3pBlu3** [comment]
Finde alles geil...kann meinetwegen auf den master. Wir sollten dann aber bald auch grunt mit einführen?!

**Comment by IvenZ** [comment]
Ich schau mal, dass wir das bis Sonntag deployed bekommen.

Muss auf den Server nur node installiert werden?

**Comment by FabiApfelkern** [comment]
Jo. Sollte noch installiert werden. Ich werde das am Samstag machen!

**Comment by IvenZ** [comment]
Bin Samstag ab ~16.30 Uhr online.
Wenn du was brauchst, sag bescheid.

**Comment by FabiApfelkern** [comment]
Node ist auf dem Server installiert. Sollte laufen.

**Comment by IvenZ** [comment]
@FabiApfelkern merge mal bitte deinen Branch mit develop, damit wir es da erst testen können.

**Comment by FabiApfelkern** [comment]
Ist erstmal drin, aber noch nicht perfekt. Die Chat.js muss noch mal angefasst werden.

**Comment by FabiApfelkern** [comment]
@De3pBlu3 Kannst du das noch fertig machen?
